06-building-a-program-start-to-finish/solving-the-problem/output/inverted-index.js

```diff
@@ -0,0 +1,102 @@
+const helpers = require('./helpers');
```

Comment:
According to AirBnB your file name should match their default export. i.e `InvertedIndex.js`